h/views/api/groups.py

```diff
@@ -160,17 +155,13 @@ def read_members(context: GroupContext, _request):
     link_name="group.member.delete",
     description="Remove the current user from a group",
     is_authenticated=True,
+    permission=Permission.Group.MEMBER_REMOVE,
 )
-def remove_member(context: GroupContext, request):
+def remove_member(context: GroupMembershipContext, request):
     """Remove a member from the given group."""
-    # Currently, we only support removing the requesting user
-    if request.matchdict.get("userid") == "me":
-        userid = request.authenticated_userid
-    else:
-        raise HTTPBadRequest('Only the "me" user value is currently supported')
```

Comment:
`me` is handled by the context factory now and has already been resolved before the view is called (see above).